kefir.js.flow

```diff
@@ -0,0 +1,158 @@
+  flatMapLimit<V2,E2>(transform: (value: V) => Observable<V2,E2>, limit: number): Observable<V2,E|E2>;
+  flatMapErrors<V2,E2>(transform: (error: E) => Observable<V2,E2>): Observable<V|V2,E2>;
+
+  filterBy(otherObs: Observable<any,any>): Observable<V,E>;
+  skipUntilBy(otherObs: Observable<any,any>): Observable<V,E>;
+  takeUntilBy(otherObs: Observable<any,any>): Observable<V,E>;
+  bufferBy(otherObs: Observable<any,any>, options?: {flushOnEnd?: boolean}): Observable<V[],E>;
+  bufferWhileBy(otherObs: Observable<any,any>, options?: {flushOnEnd?: boolean, flushOnChange?: boolean}): Observable<V[],E>;
+  sampledBy(otherObs: Observable<any,any>): Observable<V,E>;
+  sampledBy<V2,V3>(otherObs: Observable<V2,any>, combinator: (obsValue: V, otherObsValue: V2) => V3): Observable<V3,E>;
+  bufferWithTimeOrCount(time: number, count: number, options?: {flushOnEnd: boolean}): Observable<V,E>;
+}
+
+declare class Pool<V,E=*> extends Observable<V,E> {
+  plug(s: Observable<V,E>): () => void;
+}
+
+declare class Stream<+V,+E=*> extends Observable<V,E> {
+}
+
+declare class Property<+V,+E=*> extends Observable<V,E> {
+}
+
+declare var Kefir: {
+  Observable: typeof Observable;
+  Pool: typeof Pool;
+  Stream: typeof Stream;
+  Property: typeof Property;
+
+  staticLand: {
+    Observable: {
+      ap<A,B,E1,E2>(obsF: Observable<(x: A) => B, E1>, obsV: Observable<A, E2>): Observable<B,E1|E2>;
+      bimap<V1,E1,V2,E2>(fnE: (x: E1) => E2, fnV: (x: V1) => V2, obs: Observable<V1,E1>): Observable<V2,E2>;
+      chain<V,V2,E,E2>(cb: (value: V) => Observable<V2,E2>, s: Observable<V,E>): Observable<V2,E|E2>;
+      concat<V,E>(obss: Observable<V,E>[]): Observable<V,E>;
+      empty(): Observable<*,*>;
+      map<V,V2,E>(cb: (value: V) => V2, s: Observable<V,E>): Observable<V2,E>;
+      of<V>(value: V): Observable<V,*>;
+    };
+  };
+
+  never(): Observable<*>;
+  later<V>(delay: number, value: V): Observable<V,*>;
+  interval<V>(interval: number, value: V): Observable<V,*>;
+  sequentially<V>(interval: number, values: V[]): Observable<V,*>;
+  fromPoll<V>(interval: number, f: () => V): Observable<V,*>;
+  withInterval<V>(interval: number, f: (emitter: Emitter<V>) => void): Observable<V,*>;
+  fromCallback<V>(f: (cb: (value: V) => void) => void): Observable<V,*>;
+  fromNodeCallback<V,E>(f: (cb: (err: E, value: ?V) => void) => void): Observable<V,E>;
+  fromEvents(target: Object, eventName: string, transformer?: (event: any) => any): Observable<any,*>;
+  stream<V,E>(subscribe: (emitter: Emitter<V,E>) => ?() => void): Observable<V,E>;
+
+  constant<V>(value: V): Property<V,*>;
+  constantError<E>(err: E): Property<*,E>;
+  fromPromise<V>(promise: Promise<V>): Property<V,*>;
+
+  fromESObservable(observable: Object): Observable<any,*>;
+  toESObservable(): Object;
+
+  combine<E>(obss: Observable<any,E>[], passiveObss?: Observable<any,E>[], combinator?: Function): Observable<any,E>;
```

Comment:
Ok, let's use any for now and see how it goes.

But we still need 3 overloads because currently we wouldn't be able to pass a function as second argument, right?